lambda-functions/Lambda/SQLAlchemy/db_utils.py

```diff
@@ -0,0 +1,11 @@
+from subprocess import call
+
+# on EC2 instance with AWS CLI configured for my user
+# db_host = call("aws rds describe-db-instances | jq -r '.DBInstances[]|select(.DBInstanceIdentifier=='rds-reddit-data').Endpoint|.Address'", shell=True)
```

Comment:
Instead of having this as a comment, consider creating a `scripts` directory in the root of this repo, and include the above command as an executable script available there.